fetch-lorempixel-images.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# Demo use only. Images from lorempixel, all rights [p]reserved.
+for i in `seq 10` ; do curl -s -H "Accept: image/jpg" http://lorempixel.com/400/400/nature/$i/ >$i.jpg ; done
```

Comment:
It can not be moved to that folder, since it would break serving, since serving depends on file types being MIME-supported :)

The latter is a very good suggestion. PTAL at dd7da1d. Thanks!